homeassistant/components/zabbix/__init__.py

```diff
@@ -129,7 +132,7 @@ def event_to_metrics(
                 _state_as_value = float(state_helper.state_as_number(state))
                 floats[entity_id] = _state_as_value
             except ValueError:
-                strings[entity_id] = state.state
+                strings[entity_id] = str(state.state)
```

Comment:
I don't think we should waste cycles on the stringification here considering the default value of `include_strings` is `False`.
Maybe do:
```suggestion
                if include_strings:
                    strings[entity_id] = str(state.state)
```